src/FormBuilder/resources/js/form-builder/src/settings/design/index.tsx

```diff
@@ -153,7 +157,9 @@ const FormDesignSettings = () => {
                                 dispatch(setFormSettings({multiStepFirstButtonText}));
                                 publishSettings({multiStepFirstButtonText});
                             }}
-                            help={__('Customize the text that appears in the first step, prompting the user to go to the next step.')}
+                            help={__(
+                                'Customize the text that appears in the first step, prompting the user to go to the next step.'
```

Comment:
This needs the "give" domain

Reply:
Resolved: 6a678012044cb306497b3b1453304db45a0e080b